.github/workflows/benchmarks.yml

```diff
@@ -0,0 +1,31 @@
+name: Compute Benchmarks
```

Comment:
teeny, tiny thing to consider: for UMF we don't execute "Compute Benchmarks" - we execute our benchmarks written with gtest. We could consider renaming it (in all places, incl. writing comments, etc...)